bundles/org.openhab.binding.revogismartstripcontrol/src/main/java/org/openhab/binding/revogismartstripcontrol/internal/RevogiSmartStripControlBindingConstants.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.revogismartstripcontrol.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link RevogiSmartStripControlBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Andi Bräu - Initial contribution
+ */
+@NonNullByDefault
+public class RevogiSmartStripControlBindingConstants {
+
+    private static final String BINDING_ID = "revogismartstripcontrol";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID SMART_STRIP_THING_TYPE = new ThingTypeUID(BINDING_ID, "smartstrip");
+
+    // List of all Channel ids
+    public static final String PLUG_1_SWITCH = "plug1#switch";
+    public static final String PLUG_2_SWITCH = "plug2#switch";
+    public static final String PLUG_3_SWITCH = "plug3#switch";
+    public static final String PLUG_4_SWITCH = "plug4#switch";
+    public static final String PLUG_5_SWITCH = "plug5#switch";
+    public static final String PLUG_6_SWITCH = "plug6#switch";
+    public static final String ALL_PLUGS = "overallPlug#switch";
```

Comment:
Why do you need a channel for all the plugins? Couldn't users create this themselves with an item group?

Reply:
because it is a power plug with 6 power outlets, see https://www.revogi.com/wp-content/themes/salem/images/icon/smart-power-strip-eu-b.png

and the device to switch all plugs together (the ALL_PLUGS switch)